setup.cfg

```diff
@@ -22,14 +22,16 @@ classifiers =
     Programming Language :: Python :: 3.7
```

Comment:
```suggestion
```

[Python 3.7 is end-of-life](https://devguide.python.org/versions/) so we should remove it -- **stactools** is >= 3.8.